CONTRIBUTING.md

```diff
@@ -0,0 +1,101 @@
+# Contributing to InteractiveSideMenu
+
+:+1: First of all, big thanks for your desire to contibute in this project :+1:
+
+## How to contribute
+
+The easiest way to contribute is to *report us about a bug* or *propose a new feature*. Use [GitHub Issues](https://github.com/handsomecode/InteractiveSideMenu/issues) to say us about it.
+Please, feel free to send your suggestions concerning any library issue, documentation and ReadMe typos, [Sample](./Sample) improvements, and any other enhancement. **We really appreciate your participation!**
```

Comment:
I think we should use capital letters for such kind of words like README, CHANGELOG, LICENSE and CONTRIBUTING.